clients/client-python/tests/integration/integration_test_env.py

```diff
@@ -80,6 +80,12 @@ def setUpClass(cls):
             )
             sys.exit(0)
 
+        # remove data dir under gravitino_home
+        data_dir = os.path.join(cls.gravitino_home, "data")
+        if os.path.exists(data_dir):
+            logger.info("Remove Gravitino data directory: %s", data_dir)
+            subprocess.run(["rm", "-rf", data_dir], check=False)
```

Comment:
It's better to use `shutil.rmtree` to delete the dir recursively.

Reply:
done